packages/api-explorer/style/markdown-overrides.scss

```diff
@@ -24,7 +24,7 @@
         color: #bbbec1;
         p { color: inherit !important }
         > .CodeTabs, > pre {
-          filter: invert(.9) !important;
+          filter: var(--md-code-text, invert(0.9)); /* ¡hacksaurz! */
```

Comment:
lol this just turns off my shitty-ass dark theme if the user customizes the code block theme 😉